r-pkg/R/es_search.R

```diff
@@ -85,7 +85,7 @@ es_search <- function(es_host
 ){
 
     # Check if this is an aggs or straight-up search query
-    if (length(query_body) > 1 || ! "character" %in% class(query_body)){
+    if (assertthat::is.string(query_body)){
```

Comment:
this assertion was the problem with Travis! I should have made it `!is.string()` and I made it `is.string()`. Good for us for having good tests!

Reply:
yeah this is working now 😄